docs/api/create-list.md

```diff
@@ -1,66 +1,73 @@
+| `hooks`         | `Object`                            | `{}`                          | Specify hooks to execute functions after list actions.         |
+| `labelField`    | `String`                            | `name`                        | Specify a field to use as a label for individual list items.   |
+| `labelResolver` | `Function`                          | Resolves `labelField` or `id` | Function to resolve labels for individual list items.          |
+| `access`        | `Function` \| `Object` \| `Boolean` | `true`                        | [Access control](/guides/access-control) options for the list. |
+| `adapterConfig` | `Object`                            |                               | Override the adapter config options for a specific list.       |
+| `itemQueryName` | `String`                            |                               | Changes the _item_ name in GraphQL queries and mutations.      |
+| `listQueryName` | `String`                            |                               | Changes the _list_ name in GraphQL queries and mutations.      |
+| `singular`      | `String`                            |                               | Specify a singular noun for Keystone to use for the list.      |
+| `plural`        | `String`                            |                               | Specify a plural for Keystone to use for the list.             |
+| `path`          | `String`                            |                               | Changes the path in the Admin UI.                              |
+| `plugins`       | `Array`                             | `[]`                          | An array of `plugins` that can modify the list config.         |
 
-A singular name for the list e.g. 'Post'. Where possible plural names will be generated automatically or can be configured in options.
+### `fields`
 
-## options
+Defines the fields to use in a list.
 
-### fields
+#### Usage
 
-#### defaultValue
-
-If not set a value must be provided when creating new items.
-
-For a 'nullable' field, set `defaultValue: null`.
-
-May be a static value, or a function which returns the value / a Promise for the value.
-
-NOTE: Some types have a pre-configured defaultValue;
-
-- **Relationship**: `null`
+```javascript
+keystone.createList('Post', {
+  fields: {
+    title: { type: Text },
+  },
+});
+```
 
-NOTE: This can have impacts when combined with [access control](./access-control.md)
-where it's possible a user is never able to create new items because they don't
-have access to create a certain field, which attempts to use the `defaultValue`,
-but then throws an error because no `defaultValue` has been set.
+See: [Fields](/keystone-alpha/fields/) for more information on configuring field options.
 
-### schemaDoc
+### `schemaDoc`
 
-A description for the list. Used in the Admin UI.
+A description for the list used in the AdminUI.
```

Comment:
```suggestion
A description for the list used in the GraphQL schema.
```

This isn't actually used in the Admin UI, whoops!

Reply:
It should be 🤣